docs/home/howto/CLUSTER.md

```diff
@@ -795,13 +795,14 @@ Epiphany has the ability to automaticly scale and cluster certain components on
 
 Then when applying the changed configuration using Epicli additional VM's will be spawned and configured or removed. The following components support scaling/clustering:
 
+- repository: In standard Epiphany deploment you need just one repository VM. Scaling up repository component will create a new standalone VM. Scaling down will complete remove it (LIFO - Last In, First Out). However even if you create more then one VM, by default all other components will use the first one.
 - kubernetes_master: When increased this will setup additional control plane nodes, but in the case of non-ha k8s cluster, existing control plane node must be promoted first. At the moment there is [no ability](https://github.com/epiphany-platform/epiphany/issues/1579) to downscale.
 - kubernetes_node: When increased this will setup additional nodes with `kubernetes_master`. There is [no ability](https://github.com/epiphany-platform/epiphany/issues/1580) to downscale.
 - ignite
 - kafka: When changed this will setup or remove additional nodes for the Kafka cluster. Note that there is an [issue](https://github.com/epiphany-platform/epiphany/issues/1576) that needs to be fixed before scaling usage.
-- load_balancer
+- load_balancer - Scaling up repository component will create a new standalone VM. Scaling down will complete remove it (LIFO - Last In, First Out).
```

Comment:
completely

Reply:
Maybe we should mentione something about that second load balancer machine will be created, but it will not create load balancer's cluster by default